### ISSUE TITLE
Error: "Failed to get device group list when searching for [Device Group]" - Can't install Argus

### ISSUE THREAD

**Issue by rsryanstephen** [logicmonitor/k8s-argus#73]
-   **Argus Version: 0.1.0**
-   **Kubernetes Version: 1.10.0**
-   **Docker Version:**
-   **Helm Version: 2.9.0-rc3**

Hi guys,
While trying to install Argus, we are having trouble with the specified "CLUSTER_NAME" field.

Log back from Argus deployment:
	time="2018-05-22T08:53:23Z" level=fatal msg="Failed to get device group list when searching for \"Kubernetes Cluster: Homestead Prod Servers\": [REST] [0] "

Include this as the create command:

#!/bin/bash

NAMESPACE="kube-public"
RELEASE_NAME="logicmonit-argus"
ACCESS_ID="**"
ACCESS_KEY="**"
ACCOUNT="ryan.stephen@structureit.net"
CLUSTER_NAME="Homestead Prod Servers"
COLLECTOR_DESCRIPTION="ip-173-31-22-76.ec2.internal"
COLLECTOR_VERSION="26.001"

helm install -f config.yaml logicmonitor/argus \
    --debug \
    --wait \
    --kube-context hs-production.k8s.local \
    --name "$RELEASE_NAME" \
    --set global.accessID="$ACCESS_ID" \
    --set global.accessKey="$ACCESS_KEY" \
    --set global.account="$ACCOUNT" \
    --set clusterName="$CLUSTER_NAME" \
    --set collectorDescription="$COLLECTOR_DESCRIPTION" \
    --set collectorVersion="$COLLECTOR_VERSION"

--------------------------------------
I've included screenshots of the device associated with the given Collector Description and Device Group, as well as the "Kubernetes Cluster: Homestead Prod Servers" Device Group.

![screenshot - device associated with the given collector description and device group](https://user-images.githubusercontent.com/30828027/40355505-e138746e-5db6-11e8-9faa-b32f4e1d2232.png)
![screenshot - device group - kubernetes cluster homestead prod servers](https://user-images.githubusercontent.com/30828027/40355516-e9c0b222-5db6-11e8-8781-4676ef50c97f.png)

**Comment by andrewrynhard** [comment]
@rsryanstephen I noticed that you used your email address for the `ACCOUNT `. That should be the name of the account you use to log in to LogicMonitor. For example, if the login to `https://company.logicmonitor.com`, you would use `company` for the value of `ACCOUNT`.

**Comment by rsryanstephen** [comment]
Thanks Andrew, I will change that and get back to you if we have further
trouble.

Kind Regards

**Comment by rsryanstephen** [comment]
Hi Andrew, we have changed the necessary config variables and it appears
to have installed correctly with feedback from the terminal attached as a
screenshot.

There seems to be nothing wrong now except for the fact that no Device
groups were created in our Logic Monitor account as indicated by the
documentation:
https://logicmonitor.github.io/k8s-argus/docs/device-tree-management/

Perhaps we can set up a video call with you to address these issues, as I
feel there is a disconnect between what we are expecting and what is
provided.

I considered Thursday (tomorrow) at 10:00 California time?

![image](https://user-images.githubusercontent.com/
30828027/40704860-a8ea9284-63e9-11e8-8cc8-b9d08fe0ddbe.png)

**Comment by andrewrynhard** [comment]
@rsryanstephen My apologies for not getting back to you sooner. Would you be able to email me and we can set something up? I would suggest contact your account manager and going through them.

**Comment by rsryanstephen** [comment]
No problem Andrew, Thanks for getting back to us.
I believe our Account Manager is Anna Cecchin:
anna.cecchin@logicmonitor.com

I could possibly set something up through her, but otherwise if you're
available we could have a zoom chat on Tuesday at 9:00 your time?

Let me know if it's ok.

Thanks

**Comment by andrewrynhard** [comment]
@rsryanstephen sure that works for me.

**Comment by rsryanstephen** [comment]
You are invited to a Zoom meeting now.

Join from PC, Mac, Linux, iOS or Android: https://zoom.us/j/118358268

Or iPhone one-tap:
    US: +14086380986,,118358268#  or +16465588665,,118358268#
Or Telephone:
    Dial(for higher quality, dial a number based on your current location):
        US: +1 408 638 0986  or +1 646 558 8665
    Meeting ID: 118 358 268
    International numbers available: https://zoom.us/u/emfv1JTmp

**Comment by rsryanstephen** [comment]
Hi Andrew,

Sorry I made a hash-up with the time zones!
It appears that our scheduled meeting time with you is at 3:00 in the
morning for us.
What I actually needed to do was to make it 18:00 our time (Johannesburg)
which would be 9:00 your time (Santa Barbara).

We will have to reschedule for another time, basically any day between 8:00
- 10:00 your time.
I just need to ask which day would suit you?
